src/main/java/sirius/biz/storage/layer3/VirtualFile.java

```diff
@@ -1433,6 +1433,12 @@ private String parsePathFromUrl(URL url, Predicate<String> fileExtensionVerifier
 
             if (Strings.isEmpty(path) && !url.getPath().equals(lastConnectedURL.getPath())) {
                 // We don't have a path yet but we followed redirects so we check the new URL
+                if (headRequest.getResponseCode() == 404 && lastConnectedURL.toString().contains("Ã")) {
```

Comment:
use a constant here ? I think HttpUrlConntections Or HttpResponseState or the like defiens them

Reply:
maybe also add the hint why we check agains "A" in a short inline comment to clarify the intention...